lore/estimators/sklearn.py

```diff
@@ -63,6 +65,23 @@ class Regression(Base):
 
 
 class BinaryClassifier(Base):
+    def __init__(self, estimator):
+        super(BinaryClassifier, self).__init__(estimator, eval_metric='logloss', scoring_metric='auc')
+
+    @before_after_callbacks
+    @timed(logging.INFO)
+    def evaluate(self, x, y):
+        import sklearn
```

Comment:
This can go in the global scope at the top of the file since it already `require(lore.dependencies.SKLEARN)`.